Makefile

```diff
@@ -132,6 +135,10 @@ compile-requirements: $(COMMON_CONSTRAINTS_TXT) ## Re-compile *.in requirements
 		pip-compile -v --no-emit-trusted-host --no-emit-index-url $$REBUILD ${COMPILE_OPTS} -o $$f.txt $$f.in || exit 1; \
 		export REBUILD=''; \
 	done
+
+	pip install -qr requirements/edx/pip.txt
```

Comment:
Should we do this step before compiling requirements or after...?
Looking at the reference [PR ](https://github.com/openedx/edx-repo-health/pull/271/files), we install these again before compiling requirements.

Reply:
We do this before compiling other requirements to save execution time in case of failure but in the current case, our `pip-tools.txt` gets compiled with other requirement files in the loop so we can't move this before the compilation. 
We need to test installing both `pip` and `pip-tools` after compilation so we have to go this way.